homeassistant/components/climate/__init__.py

```diff
@@ -290,15 +247,20 @@ def target_humidity(self):
         return None
 
     @property
-    def current_operation(self):
-        """Return current operation ie. heat, cool, idle."""
+    def operation_mode(self):
+        """Return current operation ie. heat, cool."""
```

Comment:
"""Return operation mode ie. heat, cool.""" lets stick to "operation mode" and "current operation" as the two discrete data points.